app/src/org/commcare/android/database/user/models/FormRecord.java

```diff
@@ -133,6 +149,12 @@ public FormRecord updateInstanceAndStatus(String instanceURI, String newStatus)
         return fr;
     }
 
+    public static FormRecord StandInForDeletedRecord() {

```

Comment:
Why is the first letter capitalized?

Reply:
I thought that was the java convention for static methods that serve as constructors. If you think I'm making that up, let me know...